### DISCUSSION TITLE
Laravel Trademark

### DISCUSSION THREAD

**Discussion by andrey-helldar** [Laravel-Lang/common#11] [Q&A]
![image](https://user-images.githubusercontent.com/10347617/216400624-01c80db1-85a0-42d7-8dfb-86adcf9b6ecb.png)

@Laravel-Lang/laravel-lang

Taylor posted a new section on the official website today - https://laravel.com/trademark

According to it, we must either change the name of "Laravel Lang" to something else, for example, "Lang for Laravel", or get official permission.

One way or another, the name of the project was called into question.

In the near future, I'm going to write to the Laravel team about our situation in order to find the best option.

I want to do this because before that I contacted them many times about obtaining translation keys for the Spark and Nova projects. Thus, they are aware of our existence.

**Comment by mohamedsabil83** [comment]
I agree with contacting them first before deciding on the next step. If they give us permission to keep the name will be good. In the meantime, we can start thinking about choices for the new name in case they refuse.

**Reply by EuCarlos** [reply]
I agree with every single word @mohamedsabil83 said.

**Reply by andrey-helldar** [reply]
A few minutes ago I sent an email to Taylor. Now we are waiting for an answer.

I think that even in case of refusal, they will give us some time to change the name, so we can not rush.

From the simplest, we can consider the option "Lang for Laravel" ¯\_(ツ)_/¯

**Reply by caouecs** [reply]
if we must change, we change

**Reply by andrey-helldar** [reply]
@Laravel-Lang/laravel-lang, hey! I have great news! We were allowed to use the name! 🤝

![image](https://user-images.githubusercontent.com/10347617/216442568-d5db79ac-2532-498a-8930-80e4934fc2b9.png)

**Reply by mohamedsabil83** [reply]
That's great 👍🏽

**Reply by luisprmat** [reply]
Great! We are a recognized project by Laravel team!

**Reply by makowskid** [reply]
That's great news! Our small project becomes semi-native to Laravel ecosystem thanks to it :)

ALSO: for any future repositories/projects we can use a combination of "Lara+" or similar (LaraLang, LaraTranslate, LangVel etc). Should not violate the trademark.

**Reply by andrey-helldar** [reply]
@makowskid, part of the word is also a trademark. Taylor wrote about it:

> We would strongly discourage, and likely would consider to be problematic, a name such as LaravelMan, Laravel Management, LaraMan, etc.

But if Taylor approves, we can use 🙂

**Reply by masterix21** [reply]
It sounds good :)
